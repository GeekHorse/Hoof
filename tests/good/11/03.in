new right one done
new in subone done
new in subsubone done
new out subtwo done
new out two done
save
