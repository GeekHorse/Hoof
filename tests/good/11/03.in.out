new right one done
new in subone done
new in subsubone done
out
new down subtwo done
out
new down two done
